cmake/kokkos_functions.cmake

```diff
@@ -314,6 +314,151 @@ MACRO(kokkos_create_imported_tpl NAME)
   ENDIF()
 ENDMACRO()
 
+#
+# @MACRO: KOKKOS_FIND_HEADER
+#
+# Function that find a particular header. This searches custom paths
+# or default system paths depending on options. In constrast to CMake
+# default, custom paths are prioritized over system paths
+#
+# Usage::
+#
+#   KOKKOS_FIND_HEADER(
+#     <VAR_NAME>
+#     <HEADER>
+#     <TPL_NAME>
+#     ...
```

Comment:
You forgot to close the parentheses.

Also the signature reads like this
```
KOKKOS_FIND_HEADER(
    <VAR>
    <HEADER>
    <TPL>
    [ALLOW_SYSTEM_PATH_FALLBACK]
    [PATHS path1 [path2 ...]]
)
```

Reply:
Done.